configure

```diff
@@ -77,12 +80,24 @@ RCOBJS=
 STRIP=
 ARCHS=
 PC_CFLAGS=
-prefix=${prefix-/usr/local}
+if test "${uname}" = "Haiku" || test "${uname}" = "haiku"; then
+  prefix=${prefix-/boot/system}
+else
+  prefix=${prefix-/usr/local}
+fi
 exec_prefix=${exec_prefix-'${prefix}'}
 bindir=${bindir-'${exec_prefix}/bin'}
-libdir=${libdir-'${exec_prefix}/lib'}
+if test "${exec_prefix}" = "/boot/system" || (test "${prefix}" = "/boot/system" && test "${exec_prefix}" = '${prefix}'); then
```

Comment:
This check doesn't work like this. Packages use a prefix like `/packages/$packageName/.self` (which at runtime is a symlink to where the package is installed, usually `/boot/system`), or it is a `non-packaged` path (see above).

The same applies also everywhere else where you check the prefix.

Reply:
Can't just evaluate `exec_prefix` because it isn't always variable reference, hence single quotes, it can be a literal path. When packaging, `prefix` gets overwritten.